kokoro/android-release/build.sh

```diff
@@ -29,38 +29,34 @@ TARGET_ARCH=$1
 # Get NINJA.
 wget -q https://github.com/ninja-build/ninja/releases/download/v1.7.2/ninja-linux.zip
 unzip -q ninja-linux.zip
-export PATH="$PWD:$PATH"
-export BUILD_NDK=ON
-export ANDROID_NDK=/opt/android-ndk-r15c
-git clone --depth=1 https://github.com/taka-no-me/android-cmake.git android-cmake
```

Comment:
yikes.  yep, that's no longer supported now that Android builds support Cmake.  The supporting toolchain file is in the NDK now, as you discovered.